application_store/db/models/application/applications.py

```diff
@@ -42,7 +42,11 @@ class Applications(BaseModel):
     __table_args__ = (db.UniqueConstraint("fund_id", "round_id", "key", name="_reference"),)
 
     def as_dict(self):
-        date_submitted = self.date_submitted.isoformat() if self.date_submitted else "null"
+        date_submitted = (
+            (self.date_submitted if type(self.date_submitted) is str else self.date_submitted.isoformat())
```

Comment:
this feels odd - I think we should have this always as a datetime instance - do you know where the string version is coming from?

Reply:
sorry yep ignore me - we still are reading from the queue for one deployment 👍